### ISSUE TITLE
QueryException bug

### ISSUE THREAD

**Issue by PHPhantom** [stwe/DatatablesBundle#679]
When using DatatablesBundle Exception is thrown:

> [Syntax Error] line 0, col -1: Error: Expected IdentificationVariable | ScalarExpression | AggregateExpression | FunctionDeclaration | PartialObjectExpression | "(" Subselect ")" | CaseExpression, got end of string.

This is happening because of missing buildQuery() on datatableQueryBuilder before execute

https://github.com/stwe/DatatablesBundle/blob/ddc2cb3af707d46ba00cd480ca222378aaf80c77/Response/DatatableResponse.php#L136

To solve the issue the code should be like this:

```php
$paginator = new Paginator($this->datatableQueryBuilder->buildQuery()->execute(), $fetchJoinCollection);
```

Symfony version - 3.3.6

**Comment by Seb33300** [comment]
What is the code to reproduce the error?

**Comment by PHPhantom** [comment]
@Seb33300 I used from https://github.com/stwe/DatatablesBundle/blob/master/Resources/doc/installation.md

```php
public function indexAction(Request $request)
{
    $isAjax = $request->isXmlHttpRequest();

    // Get your Datatable ...
    //$datatable = $this->get('app.datatable.post');
    //$datatable->buildDatatable();

    // or use the DatatableFactory
    /** @var DatatableInterface $datatable */
    $datatable = $this->get('sg_datatables.factory')->create(PostDatatable::class);
    $datatable->buildDatatable();

    if ($isAjax) {
        $responseService = $this->get('sg_datatables.response');
        $responseService->setDatatable($datatable);
        $responseService->getDatatableQueryBuilder();

        return $responseService->getResponse();
    }

    return $this->render('post/index.html.twig', array(
        'datatable' => $datatable,
    ));
}
```

**Comment by Seb33300** [comment]
This is because I made changes in master and now, `buildQuery()` is not required anymore.

I just published v1.0.2 to fix this.

If you want to keep v1.0.1, you can look at the v1.0.1 documentation:
https://github.com/stwe/DatatablesBundle/blob/v1.0.1/Resources/doc/installation.md

**Comment by PHPhantom** [comment]
Cool! Thanks!
